core/Cargo.toml

```diff
@@ -12,6 +12,7 @@ license = "MIT OR Apache-2.0"
 cherry-evm-validate = { path = "../evm-validate", version = "0.1.0" }
 cherry-evm-decode = { path = "../evm-decode", version = "0.1.0" }
 cherry-cast = { path = "../cast", version = "0.1.0" }
+cherry-svm-decode = { path = "../svm-decode", version = "0.0.1" }
```

Comment:
better to start from 0.1.0 because need to bump version with every release if we set it to 0.0.1. With 0.1.0 we have opportunity to just bump it to 0.1.x if we don't make any breaking changes